index.js

```diff
@@ -1,56 +1,96 @@
-const v = require('node-bindgen-loader')({
-  moduleName: 'ssb-validate2-rsjs-node',
-  dir: __dirname
-})
+const v = require("node-bindgen-loader")({
+  moduleName: "ssb-validate2-rsjs-node",
+  dir: __dirname,
+});
 
-const stringify = (msg) => JSON.stringify(msg, null, 2)
+const stringify = (msg) => JSON.stringify(msg, null, 2);
 
-const verifySignatures = (msgs, cb) => {
-  if (!Array.isArray(msgs)) return "input must be an array of message objects";
+const verifySignatures = (hmacKey, msgs, cb) => {
+  if (!Array.isArray(msgs)) {
+    cb(new Error("input must be an array of message objects"));
+    return;
+  }
   const jsonMsgs = msgs.map(stringify);
-  const [err, result] = v.verifySignatures(jsonMsgs);
+  // convert `null` and `undefined` to a string ("none") for easier matching in rustland
+  if (!hmacKey) hmacKey = "none";
+  const [err, result] = v.verifySignatures(hmacKey, jsonMsgs);
+  if (err) {
+    cb(new Error(err), result);
```

Comment:
It's mostly harmless to include `result` here, but usually in Node.js we don't, we just call `cb(new Error(err));`

Reply:
Good to know! I'll make the change. I'd like for my JS code to be as idiomatic as possible, so I really appreciate this kind of feedback.